src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -600,7 +600,7 @@ public static bool IsLocalFunctionDeclarationContext(this SyntaxTree syntaxTree,
                 if (container is IncompleteMemberSyntax incompleteMember)
                     return incompleteMember.Type.IsKind(SyntaxKind.RefType);
 
-                if (container is CompilationUnitSyntax or BaseNamespaceDeclarationSyntax or TypeDeclarationSyntax)
+                if (container is CompilationUnitSyntax or BaseNamespaceDeclarationSyntax or TypeDeclarationSyntax or VariableDeclarationSyntax)
```

Comment:
Why is this change needed?

Reply:
FileKeyword parses to VariableDeclarationSyntax, to avoid a false return had to add that in. I noticed it caused a fail with a struct test. I am working on that now.